<ui version="4.0" >
 <class>OBruteForceGui</class>
 <widget class="QMainWindow" name="OBruteForceGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OBruteForceGui</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" colspan="2" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QGroupBox" name="groupBox_6" >
          <property name="title" >
           <string>Usuario(s)</string>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Usuario</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLineEdit" name="OUsersLineEdit" />
           </item>
           <item row="2" column="0" >
            <widget class="QCheckBox" name="OUsersCheckBox" >
             <property name="text" >
              <string>Introducir 
Varios Usuarios</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QPushButton" name="OUsersPushButton" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Usuarios</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QGroupBox" name="groupBox" >
          <property name="title" >
           <string>OBjetivo(s)</string>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>Ip</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QSpinBox" name="OIpSpinBox1" >
             <property name="maximum" >
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="2" >
            <widget class="QSpinBox" name="OIpSpinBox2" >
             <property name="maximum" >
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="0" column="4" >
            <widget class="QSpinBox" name="OIpSpinBox3" >
             <property name="maximum" >
              <number>255</number>
             </property>
             <property name="value" >
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="0" column="5" >
            <widget class="QSpinBox" name="OIpSpinBox4" >
             <property name="maximum" >
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="4" >
            <widget class="QCheckBox" name="OMachinesNameCheckBox" >
             <property name="text" >
              <string>Introducir por nombre</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="6" >
            <widget class="QLineEdit" name="OMachinesLineEdit" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3" >
            <widget class="QCheckBox" name="OMachinesCheckBox" >
             <property name="text" >
              <string>Introducir 
Varias Maquinas</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3" colspan="3" >
            <widget class="QPushButton" name="OMachinesPushButton" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Maquinas</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item rowspan="2" row="0" column="2" >
         <widget class="QGroupBox" name="groupBox_2" >
          <property name="title" >
           <string>Sesion</string>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label_3" >
             <property name="font" >
              <font>
               <pointsize>17</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>OBrute</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_4" >
             <property name="font" >
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Icono_Id</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2" >
         <widget class="QGroupBox" name="groupBox_5" >
          <property name="title" >
           <string>Diccionario</string>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" colspan="2" >
            <widget class="QLineEdit" name="ODictionaryLineEdit" />
           </item>
           <item row="0" column="2" >
            <widget class="QToolButton" name="ODictionaryToolButton" >
             <property name="text" >
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QCheckBox" name="ODictionaryCheckBox" >
             <property name="text" >
              <string>Seleccionar De la Lista</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QComboBox" name="ODictionaryComboBox" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>Protocolos</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QGroupBox" name="ODataBaseGroupBox" >
          <property name="title" >
           <string>Bases De Datos</string>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QCheckBox" name="OMySqlCheckBox" >
             <property name="text" >
              <string>MySql</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2" >
            <widget class="QCheckBox" name="ODataBaseCheckBox" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Base de Datos</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QCheckBox" name="OPostGresSqlCheckBox" >
             <property name="text" >
              <string>PostGresSql</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2" >
            <widget class="QLineEdit" name="ODataBaseLineEdit" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QCheckBox" name="OMsSqlCheckBox" >
             <property name="text" >
              <string>MsSql</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QCheckBox" name="ODataBasePortCheckBox" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Otro Puerto</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2" >
            <widget class="QSpinBox" name="ODataBaseSpinBox" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="maximum" >
              <number>65000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QGroupBox" name="ORepositoriesGroupBox" >
          <property name="title" >
           <string>Repositorios</string>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QCheckBox" name="OCvsCheckBox" >
             <property name="text" >
              <string>Cvs</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLabel" name="label_9" >
             <property name="text" >
              <string>Directorio</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" >
            <widget class="QLineEdit" name="ORepositoriesCvsLineEdit" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QCheckBox" name="OSvnCheckBox" >
             <property name="text" >
              <string>Svn</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QLabel" name="label_10" >
             <property name="text" >
              <string>Branch</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QLineEdit" name="ORepositoriesSvnLineEdit" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QGroupBox" name="OEmailGroupBox" >
          <property name="title" >
           <string>E-Mail</string>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QCheckBox" name="OPop3CheckBox" >
             <property name="text" >
              <string>Pop3</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLabel" name="label_5" >
             <property name="text" >
              <string>Modo</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="2" >
            <widget class="QComboBox" name="OEmailPop3ModeComboBox" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <item>
              <property name="text" >
               <string>NORMAL</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>AS400</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QCheckBox" name="OSnmpCheckBox" >
             <property name="text" >
              <string>Snmp</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QLabel" name="label_6" >
             <property name="text" >
              <string>TimeOut</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QLabel" name="label_7" >
             <property name="text" >
              <string>Acceso</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3" >
            <widget class="QLabel" name="label_8" >
             <property name="text" >
              <string>Version</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QSpinBox" name="OEmailSpinBox" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="value" >
              <number>2</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2" >
            <widget class="QComboBox" name="comboBox_3" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <item>
              <property name="text" >
               <string>READ*</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>WRITE</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="3" >
            <widget class="QComboBox" name="comboBox_4" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <item>
              <property name="text" >
               <string>1*</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>2C</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QCheckBox" name="OImapCheckBox" >
             <property name="text" >
              <string>Imap</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QGroupBox" name="ORemoteConnectionGroupBox" >
          <property name="title" >
           <string>Conecciones Remotas</string>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QCheckBox" name="OSshCheckBox" >
             <property name="text" >
              <string>Ssh</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QCheckBox" name="OTelnetCheckBox" >
             <property name="text" >
              <string>Telnet</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" >
            <widget class="QCheckBox" name="ORshCheckBox" >
             <property name="text" >
              <string>Rsh</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3" >
            <widget class="QCheckBox" name="ORloginCheckBox" >
             <property name="text" >
              <string>Rlogin</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QCheckBox" name="OFtpCheckBox" >
             <property name="text" >
              <string>Ftp</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QCheckBox" name="OVncCheckBox" >
             <property name="text" >
              <string>Vnc</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QLabel" name="label_11" >
             <property name="text" >
              <string>MAXSLEEP</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3" >
            <widget class="QSpinBox" name="spinBox_7" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="value" >
              <number>60</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="title" >
       <string>Informacion</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QPushButton" name="pushButton_7" >
         <property name="text" >
          <string>Log</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="pushButton_5" >
         <property name="text" >
          <string>Datos</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QPushButton" name="pushButton_2" >
         <property name="text" >
          <string>Limpiar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QGroupBox" name="groupBox_4" >
      <property name="title" >
       <string>Ejecucion</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QPushButton" name="OStopPushButton" >
         <property name="text" >
          <string>Parar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="ORunPushButton" >
         <property name="text" >
          <string>Ejecutar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2" >
     <widget class="QDockWidget" name="dockWidget" >
      <property name="font" >
       <font>
        <family>OStayPuft</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="features" >
       <set>QDockWidget::DockWidgetMovable</set>
      </property>
      <property name="windowTitle" >
       <string>Informacion Log</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QTextEdit" name="OLogPanelTextEdit" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Inicio" >
    <property name="title" >
     <string>&amp;Inicio</string>
    </property>
    <addaction name="action_Salir" />
   </widget>
   <widget class="QMenu" name="menu_Ayuda" >
    <property name="title" >
     <string>&amp;Ayuda</string>
    </property>
    <addaction name="action_Ayuda" />
    <addaction name="OActionAbout" />
   </widget>
   <addaction name="menu_Inicio" />
   <addaction name="menu_Ayuda" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Salir" >
   <property name="text" >
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext" >
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="action_Ayuda" >
   <property name="text" >
    <string>&amp;Ayuda</string>
   </property>
  </action>
  <action name="OActionAbout" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Informacion</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>OLogPanelTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>251</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel" >
     <x>246</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Salir</sender>
   <signal>activated()</signal>
   <receiver>OBruteForceGui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>281</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
